layouts/shortcodes/card.html

```diff
@@ -5,6 +5,13 @@
 {{- $image := .Get "image" }}
 {{- $context := . -}}
 
+{{/*- Adding asset support for images here, so that Hugo can do its image processing magic. -*/}}
+{{- if not (urls.Parse $image).Scheme -}}
+  {{- with resources.Get $image -}}
+    {{- $image = (.Resize "800x webp q80").RelPermalink -}}
```

Comment:
Can we make the resize target `800x`, format `webp` and quality `q80` configurable?